src/test/java/arrow/SayHelloRecipeTest.java

```diff
@@ -5,31 +5,38 @@
 import org.openrewrite.test.RewriteTest;
 
 import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.kotlin.Assertions.kotlin;
 
 class SayHelloRecipeTest implements RewriteTest {
     @Override
     public void defaults(RecipeSpec spec) {
-        spec.recipe(new SayHelloRecipe("com.yourorg.FooBar"));
+        spec.recipe(new SayHelloRecipe());
```

Comment:
You'll likely want to add Arrow to the classpath of your tests here. Without that we're unable to resolve the arrow imports & types below, and the case of missing types we back off from making further changes.

We want to be sure any changes we make are fully type attributed, so that not only includes `arrow.core.continuations.EffectScope`, but also any classes extends from, depends on, etc..

Reply:
```suggestion
        spec.recipe(new SayHelloRecipe())
          .parser(JavaParser.fromJavaVersion().classpath("arrow-core"));
```